README.md

```diff
@@ -2,20 +2,26 @@
 
 ## Installing
 
-### 1. Initialize the database
+### 1. Copy configuration
+
+Copy the `htconfig.php` to `.htconfig.php` and enter the database credentials.
+
+### 2. Initialize the database
 
 Create a database with a username and a password.
 Then import ````dfrndir.sql```` to it.
 
-### 2. Create an autoloader with composer
+	mysql -u YOURDBUSER -p YOURDBNAME < dfrnlr.sql
 
-Make sure you have composer installed globally, or rewrite the command to use a `.phar`.
+### 3. Create an autoloader with composer
+
+Make sure you have [composer](https://getcomposer.org/download/) installed globally, or rewrite the command to use a `.phar`.
```

Comment:
We ship a version of `composer.phar` in `util` as well.

Reply:
of the directory?

Reply:
Ha ha ha, I didn't realize this was the directory, never mind!